tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md

```diff
@@ -20,13 +20,13 @@ All community members are expected to adhere to our [Code of Conduct](CODE_OF_CO
 
 Information on contributing to this repository is available in our [Contributing file](CONTRIBUTING.md).
 
-When participating in {Project Name} open source community conversations and spaces, we ask individuals to follow the following guidelines:
+When participating in {{ cookiecutter.project_name }} open source community conversations and spaces, we ask individuals to follow the following guidelines:
```

Comment:
COMMUNITY_GUIDELINES.md is meant for external communities, and prolly could use a few tweaks for an innersource-style project. Comments below.